File: Aula_04/ex02.py
class Triangulo:                        # tipo de variavel
    def __init__(self):                 # todos esses metodos que tem __algumacoisa__ tem alguma funcionalidade especifica no python
        self.b = 0                      
        self.h = 0                      # campos ou atributos

    def calc_area(self):                # metodo
        return self.b * self.h / 2

x = Triangulo()
y = int()
z = 5
s = str()

print(x, type(x))
print(y, type(y))
print(z, type(z))
print(s, type(s))

File: Aula_04/ex02.c
#include <stdio.h>

typedef struct{
    double b,h;
} Triangulo;

double CalcArea (Triangulo t){
    return t.b * t.h / 2;
}

int main(){
    Triangulo x;
    x.b = 10;
    x.h = 20;
    printf("%lf",CalcArea(x));
    return 0;
}